mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -278,6 +278,11 @@ SONIC:
 		Offset: -170,0,0
 	AutoTarget:
 	WithVoxelTurret:
+	Explodes:
+		RequiresCondition: !rank-elite
+	Explodes@ELITE:
+		RequiresCondition: rank-elite
+		Weapon: UnitExplode
```

Comment:
To ensure that UnitExplode is used when rank-elite, even when the sonic tank was reloading when it got killed.